docs/run/run-sessions.mdx

```diff
@@ -108,11 +108,29 @@ with Session(service=service, backend=backend, max_time="25m"):
   ...   
 ```
 
-There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured. For premium accounts, the ITTL is 5 min, and for open plan accounts, the ITTL is 2 sec. If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see [How long a session stays active](sessions#active).
+There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured. For premium accounts, the ITTL is 1 min, and for open plan accounts, the ITTL is 2 sec. If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see [How long a session stays active](sessions#active).
 
 <span id="close"></span>
 ## Close a session 
 
+### For Qiskit 12.3 or later releases
```

Comment:
This will be `13.0` - we should also probably specify `qiskit-ibm-runtime` 13.0, not just qiskit 

same goes for the docs in [sessions.mdx](https://github.com/Qiskit/documentation/pull/87/files#diff-1ceb56b7710eae2b8dbd1b138f30a939e4f39c6f201f8421c7239afc2d0c51aaR179-R191)